CMakeLists.txt

```
@@ -10,6 +10,10 @@ set(LIBTOOL_INTERFACE 2)
 set(LIBTOOL_REVISION 3)
 set(LIBTOOL_AGE 2)
 
+# VERSION is XYYYZZ X: Major Y Minor Z PATCH
+set(GOTCHA_VERSION 100006)
```

not sure if this works, but maybe you can use a tuple here (hopefully it does not get through as a string constant):

```suggestion
set(GOTCHA_VERSION "(1, 0, 6)")
```

---

Fixed, and the tuple substitution works.